app/responses/models.py

```diff
@@ -0,0 +1,95 @@
+from django.db import models
+
+from model_utils.managers import InheritanceManager
+
+from app.feedback_forms.models import FeedbackForm
+from app.prompts.models import (
+    BinaryPrompt,
+    Prompt,
+    RangedPrompt,
+    RangedPromptOption,
+    TextPrompt,
+)
+from app.utils.models import TimestampedModelMixin, UUIDModelMixin
+
+
+class Response(TimestampedModelMixin, UUIDModelMixin):
+    feedback_form = models.ForeignKey(
+        FeedbackForm, on_delete=models.PROTECT, related_name="+"
+    )
+    url = models.TextField()
+    metadata = models.JSONField()
+
+    def __str__(self):
+        return self.url
+
+
+class PromptResponse(TimestampedModelMixin, UUIDModelMixin):
+    objects = InheritanceManager()
+    prompt_type = Prompt
+
+    response = models.ForeignKey(
+        Response, on_delete=models.PROTECT, related_name="prompt_responses"
+    )
+    prompt = models.ForeignKey(
+        Prompt, on_delete=models.PROTECT, related_name="+"
+    )
+
+    def get_subclass_prompt(self) -> Prompt:
```

Comment:
Just checking, are we making sure the naming is coherent accross the codebase? Be it subclass, specific or specialised?

Reply:
I got rid of the "specialised" terminology already.